clash-prelude/src/Clash/Signal/Bundle/Internal.hs

```diff
@@ -1,7 +1,7 @@
 {-# LANGUAGE CPP #-}
 {-# LANGUAGE TemplateHaskell #-}
 
-module Clash.Signal.Bundle.Internal where
+module Clash.Signal.Bundle.Internal (deriveBundleTuples) where
```

Comment:
Restricting imports on Internal modules is a bit of an anti-pattern IMO

Reply:
Is it?

When I did this I was thinking of it as telling GHC that all the helper functions in this module aren't used else where, so it's free to inline them in `deriveBundleTuples` and doesn't have to keep their core around.